src/containers/Layout/MainLayout.tsx

```diff
@@ -17,7 +18,7 @@ const MainLayoutWrapper = styled.div`
       padding-top: 20px;
       display: flex;
       flex-direction: column;
-      height: calc(100vh - 122px);
+      height: calc(100vh - 125px);
```

Comment:
Where does 125px come from? If it relates to another element it can be a constant

Reply:
That's temporary. Once we refactor the layout we shouldn't have to set a height at all, flex display should take care of that nicely.